openapi/version_2/paths/v2/customer_changes/customer_changes.yml

```diff
@@ -1,19 +1,94 @@
+    an existing customer record for inclusion in a customer file and onward transmission
+    to SSCL
   tags:
-    - unavailable
+  - available
   parameters:
     - $ref: '../../../../schema/parameters.yml#/regime'
   responses:
-    '200':
-      description: "Success"
-      content:
-        application/json:
-          schema:
-            type: object
+    '204':
+      description: Success
   requestBody:
     content:
       application/json:
         schema:
+          description: The address details to be updated for a given customer reference and region.
           type: object
+          properties:
+            region:
+              description: A single-digit region identifier for a transaction
+                or customer
+              type: string
+              enum:
+              - A
+              - B
+              - E
+              - N
+              - S
+              - T
+              - W
+              - Y
+              example: A
+            customerReference:
+              description: Customer Number of the invoicee
+              type: string
+              example: B19120000A
+            customerName:
+              description: Name of the customer as it should be displayed on the
+                invoice (or credit note)
+              type: string
+              maxLength: 240
+              example: Mr W Aston
+            addressLine1:
+              description: First line of billing address for a customer
+              type: string
+              maxLength: 240
+              example: Park Farm
+            addressLine2:
+              description: Second line of the billing address for a customer
+              type: string
+              maxLength: 240
+              example: Sugar Lane
+            addressLine3:
+              description: Third line of the billing address for a customer
+              type: string
+              maxLength: 240
+              nullable: true
+              example: West Waterford
+            addressLine4:
+              description: Fourth line of the billing address for a customer
+              type: string
+              maxLength: 240
+              nullable: true
+              example: Angleton
+            addressLine5:
+              description: Fifth line/town of the billing address for a customer
+              type: string
+              maxLength: 60
+              nullable: true
+              example: Southampton
+            addressLine6:
+              description: Sixth line/county of the billing address for a customer
+              type: string
+              maxLength: 60
+              nullable: true
+              example: Hampshire
+            postcode:
+              description: Postcode of billing address for a customer
+              type: string
+              nullable: true
+              example: SO74 3KD
+          required:
+          - region
+          - customerReference
+          - customerName
+          - addressLine1
```

Comment:
Argh! Don't worry about the schema for now. When we do add it, it'll be placed in `openapi/schema/schemas.yml` and the individual fields will go in `openapi/schema/fields.yml` (in fact you'll probably find most of the are already there).

The files in `paths` then reference the schema to allow for reuse. So sorry @StuAA78 I should have made the 'do not worry about schema for now' message clearer.